target/linux/ramips/dts/mt7621_ruijie_rg-ew1200g-pro.dts

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "ruijie,rg-ew1200g-pro", "mediatek,mt7621-soc";
+	model = "Ruijie RG-EW1200G PRO";
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_blue: power_blue {
+			label = "blue:power";
```

Comment:
This ```label``` property can be replaced with ```color``` and ```function``` property.

example: https://github.com/openwrt/openwrt/commit/19c45b95dbb53a7ba1a26c201277ca86293bf404